### DISCUSSION TITLE
tts慢的问题

### DISCUSSION THREAD

**Discussion by GuCangRan** [PaddlePaddle/PaddleSpeech#1280] [Q&A] [answered]
目前docker安装完成后，8c 32g 200g硬盘配置，一句话转换需要20s左右时间，这样比较慢，需要什么样的服务器配置才能提高转换速度呢？
监控发现只有单核线程在执行，提高CPU的计算能力还是加大GPU呢？这个能否给一个说明呢

**Comment by yt605155624** [comment, marked as answer]
目前命令行调用是最简单的使用方法，如果想要部署并提升资源利用率，建议您参考 examples/csmsc/tts3 里面的代码，一种比较简单的方式是，我们对于分句现在是用 for 循环最后合并，您可以尝试使用并发调用不同分句，对于声学模型和声码器，也可以尝试并发调用
